examples/STM32L475VG-DISCOVERY-IOT/BTLE_sensors_tof_demo/BTLE_sensors_tof_demo.ino

```diff
@@ -0,0 +1,427 @@
+/*
+
+ BTLE_sensors_tof_demo
+
+ This sketch provides a default demo to be used on STMicroelectronics Discovery L475VG IoT board.
+ It will use several components of the board :
+   * Low energy Bluetooth (SPBTLE_RF)
+   * temperature and pressure sensor (LPS22HB)
+   * temperature and humidity sensor (HTS221)
+   * Time Of Flight sensor (VL53L0X)
+ You can find more information on this board here :
+   http://www.st.com/en/evaluation-tools/b-l475e-iot01a.html
+
+ This sketch will launch 3 services on BLE : Acc, Environnemental and Time.
+ For testing the sketch, you can download on the playstore the "BlueNRG" application provided by
+ STMicroelectronics. (https://play.google.com/store/apps/details?id=com.st.blunrg for Android or
+ https://itunes.apple.com/fr/app/bluenrg/id705873549?mt=8 for Apple)
+ Enable Bluetooth and launch the application on your smartphone. Connect it to the BLueNRG device.
+ You will see all the services, you can click on each one.
+
+ Temperature, humidity and pressure are sent to the environment service and updated on regular
+ basis.
+ To move the cube on motion tab, you have to swipe on top of the vl53l0x sensor. To change
+ direction of the motion, use a tap. The gesture has to be done at less than 1 meter from the
+ sensor.
+
+ Do not forget to check information on the console.
+
+ */
+
+#include <SPI.h>
+#include <SPBTLE_RF.h>
+#include <sensor_service.h>
+
+#include <HTS221Sensor.h>
+#include <LPS22HBSensor.h>
+
+#include <vl53l0x_class.h>
+#include <tof_gestures.h>
+#include <tof_gestures_SWIPE_1.h>
+#include <tof_gestures_TAP_1.h>
+
+/* Configure SPI3 for BLE
+  MOSI: PC12 (44), MISO: PC11 (43), SCLK: PC10 (42)
+  */
+SPIClass SPI_3(PC12, PC11, PC10);
+
+/* Configure BTLE pins
+      csPin = PD13 (50), spiIRQ = PE6 (57), reset = PA8 (31), led = LED4
+*/
+SPBTLERFClass BTLE(&SPI_3, PD13, PE6, PA8, LED4);
+
+/* BLE device name and address */
+const char *name = "BlueNRG";
+uint8_t SERVER_BDADDR[] = {0x12, 0x34, 0x00, 0xE1, 0x80, 0x03};
```

Comment:
should the BLE address be the same for all devices ? What if several devices run in parallel in the same location ?
in the doesn't it need to be adapted for each board with the BT specific address ?